Data/collections.json

```
@@ -3677,19 +3691,18 @@
                         "ProductID":"CORINE LAND COVER (CLC)",
                         "Product_type":"STATUS",
                         "SpecificProduct":"CLC_1990, \n\n CLC_2000, \n\n CLC_2006, \n\n CLC_2012, \n\n CLC_2018",
-                        "Spatial": "Europe",
-                        "Temporal":"(*) Apr 1998 - Sep 2021",
+                        "Spatial": "Europe (*)",
                         "Catalogue":"/eodata/CLMS/Pan-European/CORINE_Land_Cover/",
                         "ProductLink":"[Details](https://land.copernicus.eu/pan-european/corine-land-cover){target='_blank'}"
                     },
                     {   
                         "ProductID":"CORINE LAND COVER (CLC)",
                         "Product_type":"CHANGES",
                         "SpecificProduct":"CHA_1990_2000, \n\n CHA_2000_2006, \n\n CHA_2006_2012, CHA_2012_2018",
-                        "Spatial": "Europe",
-                        "Temporal":"(*) Apr 1998 - Sep 2021",
+                        "Spatial": "Europe (*)",
                         "Catalogue":"/eodata/CLMS/Pan-European/CORINE_Land_Cover/",
                         "ProductLink":"[Details](https://land.copernicus.eu/pan-european/corine-land-cover){target='_blank'}"
+                        "Note": "(*) Number of involved countires (CLC_1990 – 27; CLC_2000 – 39, CLC_2006 – 39; CLC_2012 – 39; CLC 2018 – 39."
```

Issue with the symbol
```suggestion
                        "Note": "(*) Number of involved countires (CLC_1990 - 27; CLC_2000 - 39, CLC_2006 - 39; CLC_2012 - 39; CLC 2018 - 39."
                    
```